### PR TITLE
Add default values for boolean properties consumed on the advanced build property page

### PR DESCRIPTION
**Pull request by RaulPerez1**

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props [hunk 1]
```diff
@@ -81,6 +81,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DelaySign Condition="'$(DelaySign)'==''">false</DelaySign>
     <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)'==''">false</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
+    <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
```
Comment (srivatsn) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props]: Is there a way to know that this is the exhaustive list? Can you make sure to play with each property with the CPS Visible=False to make sure that it doesn't crash?

Comment (RaulPerez1) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props]: I was just talking about this with @jinujoseph, basically the properties that need defaults are those that are not strings (e.g. Booleans). These are the only 2 that I found on this bucket that needed to be default. She tried this change on top of the CPS changes and the visible property changes and things are working as expected (sans other unrelated issues).

**Review by srivatsn — APPROVED** [review]

**Review by 333fred — APPROVED** [review]

**Comment by srivatsn** [comment]
Tagging @MattGertz for approval - the template is in https://github.com/dotnet/roslyn-project-system/pull/1472 as these go together.

**Comment by srivatsn** [comment]
This was approved offline.

**Pull request merged**